force-app/main/default/triggers/AlterAmountEmployee.trigger

```diff
@@ -0,0 +1,54 @@
+// calculate the points amounts of the employees
+// Trigger is launched before process builder
+trigger AlterAmountEmployee on Task_Monitor__c ( after update) {
+	// do bulk operetions
+	List<Contact> conts=new List<Contact>();
```

Comment:
you should move this logic to a trigger handler class, and you just call the corresponding method of the class associated to the respective context variable of the trigger for each "event"

Reply:
The trigger handler class approach helps make your code more reusable and easiest to test =)